settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'codefreak-templates'
+rootProject.name = 'templates'
```

Comment:
sure that we want to do this? implication is that, in the list of dependencies in an IDE, this shows up as a generic "templates-x.x.x.jar" 🤔

Reply:
I did this for local testing so the groupId and artifactId matches `com.github.codefreak:templates` but I can revert this

Reply:
Ah, that's already the artifact name because of how JitPack works. Then I think this change is actually good. re-revert? :D

Reply:
`get reset --hard HEAD^` :sweat_smile:

Reply:
![grafik](https://user-images.githubusercontent.com/17194301/130190660-83b18462-df2d-4347-9399-bfbafd641791.png)
:D